metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -278,15 +311,17 @@ def get_workunits_internal(self) -> Iterable[MetadataWorkUnit]:
                 else:
                     self.report.report_dropped(dn)
 
-            pctrls = get_pctrls(serverctrls)
-            if not pctrls:
-                self.report.report_failure(
-                    "ldap-control", "Server ignores RFC 2696 control."
-                )
+            if serverctrls:
```

Comment:
Should the source report failure if serverctrls is not truthy ? When is serverctls not truthy ?

Reply:
> Should the source report failure if `serverctrls` is not truth ?
No, only if enabled `pagecontrol`. i fix it.

> When is serverctls not truth?
In some scenarios, clients might want to check if the server supports specific controls without actually using them. To do this, they can send an empty server control in the request and observe the server's response. If the server supports the control, it will be included in the response.

Changed on `self.lc`